docs/source/features/metrics.md

```diff
@@ -50,39 +50,40 @@ server.listen().then(({ url }) => {
 You can provide your Graph Manager API key to Apollo Server via the `ENGINE_API_KEY` environment variable. Similarly, you can assign a particular [variant](https://www.apollographql.com/docs/platform/schema-registry/#managing-environments)
 to an Apollo Server instance via the `ENGINE_SCHEMA_TAG` environment variable.
 
-You can set environment variable values on the command line as seen below, or by using the [`dotenv` npm package](https://www.npmjs.com/package/dotenv) (or similar).
+You can set environment variable values on the command line as seen below, or with the [`dotenv` npm package](https://www.npmjs.com/package/dotenv) (or similar).
 
 ```bash
 # Replace the example values below with values specific to your use case.
 ENGINE_API_KEY=YOUR_API_KEY ENGINE_SCHEMA_TAG=development node start-server.js
 ```
 
-### Client awareness
+### Identifying distinct client versions
```

Comment:
Since there's a specific configuration notion of _client version_ below, my initial thought was that the inclusion of `versions` in this title was possibly too specific.  

In other words, if `Identifying distinct clients` would be better.

Anyhow, definitely not asking for a change on this, just noting my instinct.

Reply:
Good call! Will change.